lean4-mode/lean4-mode.el

```diff
@@ -208,9 +208,30 @@ Invokes `lean4-mode-hook'.
 ;;;###autoload
 (push '("\\.lean$" . lean4-mode) auto-mode-alist)
```

Comment:
Uh, don't you want to change this one too? How did you test this patch?

Reply:
Good catch, thanks!

It is indeed poorly tested, I apologize. It is a version of something I have in my `init.el` file. In general, I don't have a systematic approach to testing my elisp code and any advice would be welcome.

Reply:
I believe Soonho was the last person to know anything about unit-testing Elisp... :)